org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "19.08",
```

Comment:
why '19.08' branch?

Reply:
it is for the electron  base app

Reply:
I think what you want is `base-version` to be that. And `branch` left unset or to `stable`